ConfigData/phase Sheet Silicate Ternary.py

```diff
@@ -1,13 +1,29 @@
 
 __author__ = 'Zack Gainsforth'
+<<<<<<< HEAD
```

Comment:
Here is another unresolved merge conflict.  You need to choose either <<<< head or >>>> 81178. and then recommit.